marklogic-data-hub-central/src/test/java/com/marklogic/hub/central/controllers/steps/IngestionStepControllerTest.java

```diff
@@ -46,4 +53,39 @@ void getIngestionSteps() throws Exception {
                 assertEquals("secondStep", array.get(1).get("name").asText());
             });
     }
+
+    @Test
+    void permittedReadUser() throws Exception {
+        installReferenceModelProject();
+        postJson(PATH + "/firstStep", newDefaultIngestionStep("firstStep"));
+
+        setTestUserRoles("hub-central-load-reader");
+
+        loginAsTestUser();
+
+        mockMvc.perform(get(PATH + "/{stepName}", "firstStep")
+                .contentType(MediaType.APPLICATION_JSON)
+                .session(mockHttpSession))
+                .andDo(result -> {
+                    MockHttpServletResponse response = result.getResponse();
+                    assertEquals(HttpStatus.OK.value(), response.getStatus());
+                });
+    }
+
+    @Test
+    void forbiddenReadUser() throws Exception {
```

Comment:
Just an FYI, there's a verifyRequestIsForbidden helper method coming in the 4820 PR. I'll see if I can get that merged into develop tomorrow so you can use it.